.github/workflows/eden.yml

```diff
@@ -16,4 +16,4 @@ jobs:
     uses: ./.github/workflows/test.yml
     with:
       eve_image: "lfedge/eve:12.4.0"
-
+    secrets: inherit
```

Comment:
As I understand from GH docs (I shared the link somewhere in discussions) in that case you should point onto secrets in nested workflows, not onto inputs.

Reply:
In this context test.yml is a nested workflow to eden.yml

Reply:
I mean you inherit secrets (no pass inputs) now. But try to use inputs in nested workflows (not secrets).

Reply:
Inside test.yml we use them as inputs like [here](https://github.com/lf-edge/eden/pull/988/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R79) test.yml is invoked by EVE repository as reusable workflow, so we should inherit secrets for test.yml, eden.yml does the same. If we are to add inputs to test.yml it'll be as it was before in EVE repository

Reply:
Any ideas how to shut yetus up on this secret keyword?

Reply:
Let's try to create `.yetus/detsecrets-ignored-hashes.txt`  with `3e26d6750975d678acb8fa35a0f69237881576b0` line inside.